reana_workflow_controller/workflow_run_manager.py

```diff
@@ -358,12 +365,42 @@ def _create_job_spec(self, name, command=None, image=None,
         if os.getenv('FLASK_ENV') == 'development':
             job_controller_env_vars.extend(
                 current_app.config['DEBUG_ENV_VARS'])
+        job_controller_env_vars.extend([{
```

Comment:
The code here and here https://github.com/reanahub/reana-job-controller/pull/158/files#diff-669c9f342cac1a455e5c652c40abf085R103 is very similar, we should create a refactor issue pointing to this code once merged.